dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/report/TrackerReportUtils.java

```diff
@@ -75,7 +76,7 @@ else if ( Date.class.isAssignableFrom( argument.getClass() ) )
         }
         else if ( Instant.class.isAssignableFrom( argument.getClass() ) )
         {
-            return DateFormat.getInstance().format( Date.from( (Instant) argument ) );
+            return DateUtils.getIso8601NoTz( DateUtils.fromInstant( (Instant) argument ) );
```

Comment:
why did you switch to `DateUtils.fromInstant`? `Date.from` seems to have the same signature

Reply:
Just because `DateUtils.fromInstant` it is null safe and it is exactly how we do it in the serializer `WriteInstantStdSerializer`